docs/source/cli/github.rst

```diff
@@ -83,7 +83,11 @@ Creates, checks out and manages GitHub PRs while keeping them reflected in branc
     If ``.git/info/milestone`` file is present, its contents (a single number --- milestone id) are used as milestone.
     If ``.git/info/reviewers`` file is present, its contents (one GitHub login per line) are used to set reviewers.
 
-    The subject of the first unique commit of the branch is used as PR title.
+    The PR title is determined in the following order of priority:
+    1. If the `--title` option is provided, its value is used as the PR title.
```

Comment:
Good spot, `--title` isn't mentioned here in the current docs

Reply:
I noticed I had neglected to add documentation and tests for gitlab. Just pushed another commit to add.